modules/lymphgen/1.0/lymphgen.smk

```diff
@@ -31,15 +31,40 @@ CFG = op.setup_module(
     subdirectories = ["inputs", "reformat_seg", "lymphgen_input", "add_svs", "lymphgen_run", "composite_other", "outputs"],
 )
 
+assert ("empty" in list(CFG["inputs"]["sample_seg"].keys())), (
```

Comment:
Is there a better way to test this that also ensures that the value is unmodified, not just present?